src/db_adapters/receipts.rs

```diff
@@ -129,6 +66,105 @@ pub(crate) async fn store_receipts(
     join!(process_receipt_actions_future, process_receipt_data_future);
 }
 
+/// Looks for already created parent transaction hash for given receipts
+async fn find_tx_hashes_for_receipts(
+    pool: &Pool<ConnectionManager<PgConnection>>,
+    receipt_ids: Vec<String>,
+    strict_mode: bool,
+) -> HashMap<String, String> {
+    let mut receipts = receipt_ids.clone();
```

Comment:
```suggestion
    mut receipt_ids: Vec<String>,
    strict_mode: bool,
) -> HashMap<String, String> {
```

Fix `receipts` -> `receipt_ids` below.